src/main/resources/reference.conf

```diff
@@ -57,6 +57,20 @@ cassandra-journal {
   # Deletes are achieved using a metadata entry and then the actual messages are deleted asynchronously
   # Number of retries before giving up
   delete-retries = 3
+  
+  # The number of retries when a read query fails.
+  read-retries = 3
+  
+  # Set this to a positive integer to enable speculative executions.
+  # The value defines the number of speculative executions that will be
+  # performed with the delay defined by 'speculative-executions-delay'.
+  # See http://docs.datastax.com/en/developer/java-driver/3.1/manual/speculative_execution/
+  # If you enable this you should also enable cassandra-query-journal.speculative-executions
+  # which is used for the relay query.
```

Comment:
yeah, those duplicate configs are not nice, eventually we will change the config structure according to https://github.com/akka/akka-persistence-cassandra/issues/81